widgetry/src/widgets/persistent_split.rs

```diff
@@ -14,35 +14,68 @@ pub struct PersistentSplit<T: Clone + PartialEq> {
 }
 
 impl<T: 'static + PartialEq + Clone + std::fmt::Debug> PersistentSplit<T> {
-    pub fn new<MK: Into<Option<MultiKey>>>(
+    pub fn widget<MK: Into<Option<MultiKey>>>(
```

Comment:
I was often confused about what I'm dealing with, so in several places in this PR I've made the names more explicit and verbose. Feel free to push back or offer alternatives.